### PR TITLE
Refactor: Migrate Styling to SCSS and Remove Material UI & TailwindCSS Dependencies

### PR DESCRIPTION
**Pull request by samiurprapon**
# 💬 Describe your changes

This PR refactors the styling approach by migrating to SCSS and removing dependencies on Material UI and `Tailwind CSS`. This change simplifies the styling stack and enhances maintainability.

Related Issue: #257 

## 📂 Area of Change

- [ ] 📱 Android
- [ ] 🖥️ Backend
- [x] 🖼️ Frontend
- [ ] 📝 Docs

## 🧬 Type of change

- [ ] 🐛 Bug fix 
- [x] ♻️ Refactor
- [ ] ✨ New feature
- [x] 📦 Dependency upgrade
- [ ] 📝 Documentation update
- [ ] 🔧 Other 

## ✨ What's New

- Migrated to SCSS for all styling to simplify and unify the codebase.
- Introduced SCSS variables and mixins for consistency in design and theming.
- Removed Unused Dependencies

## 🔒 Why

- To reduce the number of dependencies.
- To unify the styling approach, use SCSS for greater control and consistency.

## ✅ Changes

- [x] Removed `Material UI` and `Tailwind CSS`.
- [ ] Converted existing component styles to SCSS.

## 🛠️ Checklist before requesting a review

- [x] 🔍 Style guide has been followed
- [x] 🧑‍💻 Performed a self-review of my code
- [x] 🔒 No backdated packages have been added

## 💡 Additional information

- N/A

### CODE REVIEW

File: frontend/src/styles/base/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+		flex: 0%;
+		max-width: 0%;
+		-ms-flex-preferred-size: 0%;
+		-webkit-flex-preferred-size: 0%;
+	}
+	.col-#{$type}-1 {
+		flex-basis: 8.33333333%;
+		max-width: 8.33333333%;
+		-ms-flex-preferred-size: 8.33333333%;
+		-webkit-flex-preferred-size: 83.33333333%;
```
Comment (d-a-ve) [inline frontend/src/styles/base/_mixins.scss:29]: this is 83.333%, is it intentional or mistake @samiurprapon

**Comment by d-a-ve** [comment]
Also, you have 3 ts configs. And they are all the same; `tsconfig.json`, `tsconfig.node.json`, `tsconfig.app.json`. All the same content and configurations.

Is there a reason for that?? @samiurprapon

**Comment by samiurprapon** [comment]
@d-a-ve accidentally staged those files, I'm removing them.

File: frontend/src/styles/base/_mixins.scss (hunk 1 above)
Comment (samiurprapon) [reply frontend/src/styles/base/_mixins.scss]: it was a mistake, now it's been fixed

**Comment by samiurprapon** [comment]
@d-a-ve I have fixed the changes, please review it, I will merge

**Pull request merged**